PWGCF/DataModel/FemtoDerived.h

```diff
@@ -27,10 +27,12 @@ namespace o2::aod
 /// FemtoDreamCollision
 namespace femtodreamcollision
 {
-DECLARE_SOA_COLUMN(MultV0M, multV0M, float);       //! V0M multiplicity
-DECLARE_SOA_COLUMN(MultNtrPV, multNtrPV, float);   //! multiplicity of charged tracks contributing to the primary vertex
-DECLARE_SOA_COLUMN(Sphericity, sphericity, float); //! Sphericity of the event
-DECLARE_SOA_COLUMN(MagField, magField, float);     //! Sphericity of the event
+DECLARE_SOA_COLUMN(MultV0M, multV0M, float);             //! V0M multiplicity
+DECLARE_SOA_COLUMN(MultNtrPV, multNtrPV, int);           //! multiplicity of charged tracks contributing to the primary vertex
+DECLARE_SOA_COLUMN(MultNtrWithTPC, multNtrWithTPC, int); //! multiplicity of charged tracks with TPC information
+DECLARE_SOA_COLUMN(MultNtrlets, multNtrlets, int);       //! multiplicity of tracklets (used in Run2)
+DECLARE_SOA_COLUMN(Sphericity, sphericity, float);       //! Sphericity of the event
+DECLARE_SOA_COLUMN(MagField, magField, float);           //! Magnetic field of the event
```

Comment:
Are you analyzing for different multiplicity estimators?
It  looks to me that you are storing everything and deciding what to use later
This is not a good design practice